ranger-plugin/conf/ranger-servicedef-hawq.json

```diff
@@ -0,0 +1,287 @@
+{
+  "name": "hawq",
+  "implClass": "org.apache.hawq.ranger.service.RangerServiceHawq",
+  "label": "HAWQ",
+  "description": "HAWQ",
+  "guid": "1ebb27ee-549a-401d-85ab-818342ca54af",
+  "resources":
+  [
+    {
+      "itemId": 1,
+      "name": "database",
+      "type": "string",
+      "level": 10,
+      "parent": "",
+      "mandatory": true,
+      "lookupSupported": true,
+      "recursiveSupported": false,
```

Comment:
@ictmalili -- these are not well documented, we copied the definition from Hive. My understanding is that:
-- recursiveSupported would allow to define a policies as recursive which is used for HDFS access policies where a policy on a given folder, if declared as recursive, will be applied to all the files and subfolders inside a given folder. This is not applicable to us, so we do not allow creation of "recursive" policies for HAWQ.
-- excludesSupported will allow a resource specified in the policy to be "excluded". Ranger Admin UI will render the box to the right of the resource to be toggled between "included" and "excluded". I am not exactly sure what effect will that have on the policy.